README.md

```diff
@@ -2,7 +2,7 @@
 
 Joinmarket refactored to separate client and backend operations
 
-Joinmarket is coinjoin software, and includes a wallet, which requires Bitcoin Core as backend (version 0.16.3, 0.17.0 or later).
+Joinmarket is coinjoin software, and includes a wallet, which requires Bitcoin Core as backend (version 0.18.0 or later).
```

Comment:
Hmm, since when we have 0.18.0 as a requirement? Even [issue about potentially dropping just 0.16.3](https://github.com/JoinMarket-Org/joinmarket-clientserver/issues/452) is still open.

Reply:
JoinMarket (via the BitcoinCoreNoHistoryInterface) utilizes output descriptors when retrieving unspent outputs. This is only available as from Bitcoin Core v0.18.0+. It is possible to use a lower version as long as the NoHistoryInterface isn't used however so this is up for discussion on whether to recommend the use of v0.18.0+ or not. Tests will fail with v0.17.1 or lower.

Reply:
With tests it's already documented that v0.17.1 is requirement and `BitcoinCoreNoHistoryInterface` gives error to user if used on older Core versions IIRC, but other parts still work. Why I think we should keep older Bitcoin Core version support as long as possible is that big chunk of users would install bitcoind/bitcoin-qt using their Linux distributions package manager, where older version will be stable default (for example, for Gentoo it's still 0.16.3).

Reply:
I agree.